plugins/collaajUpdatePlugin/CollaajUpdatePlugin.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CollaajUpdatePlugin extends KalturaPlugin implements IKalturaServices {
+
+    const PLUGIN_NAME = "CollaajUpdate";

```

Comment:
the feature in Kaltura is named "capturespace"
I don't really mind how the API itself is called, but for the sake of consistency - keeping the same name makes sense.